docs/cli_README.md

```diff
@@ -5,21 +5,52 @@
 ### Installation:
 - Select, Download and Install latest binary from [Releases](https://github.com/fabric8-analytics/cli-tools/releases)
 
+#### curl
+
+Example:
+```
+# For Linux 64Bit, Downloading v0.0.1
+$. curl -O -L https://github.com/fabric8-analytics/cli-tools/releases/download/v0.0.1/crda_0.0.1_Linux_64bit.tar.gz
```

Comment:
Can this point to latest? we can't keep changing the version everytime.

Reply:
with the `latest`, its not forming a valid URL. tried a couple of options from https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8 but, it's kind of hackish